src/include/migraphx/op/pointwise.hpp

```diff
@@ -41,6 +41,8 @@ struct pointwise
 
     shape compute_shape(const std::vector<shape>& inputs, std::vector<module_ref> mods) const
     {
+        check_shapes(inputs, *this, true).has_at_least(1);
```

Comment:
```suggestion
        check_shapes(inputs, *this).has_at_least(1);
```
`true` is for dynamic shapes.

Reply:
Oh, I thought it was inclusive of both static and dynamic outcomes. I'll fix